lab-kyle/test/test-fs-function.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
This is a good unit test :), since the assignment involves reading data and returning multiple files' data in series I suggest you add an additional integration test to make sure your bufferfiles function succeeds in that regard as well.